openstack/cbr/v3/cbrtags/results.go

```diff
@@ -0,0 +1,41 @@
+package cbrtags
```

Comment:
better to use tags name for packge

Reply:
We can not use tags name for this package. It conflicts with the name of another package.

Reply:
alias for import didn't helps?